Add plugin Vocabulary Cards

meniam opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/meniam/obsidian-vocabulary-cards

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [x]  Linux
  - [x]  Android _(if applicable)_
  - [x]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
Please remove the `main.js` from the repo and add it to your .gitignore - compiled release files shouldn't get mixed with source code in your commits and should go to the releases files instead.

[Copyright (c) 2021 nnshi-s, Euegene Myazin](https://github.com/meniam/obsidian-vocabulary-cards/blob/11d993d81717fe284e91ed3353d1d4fd29a767a4/LICENSE#L3)
Fill this with your own info.

[plugin.app.vault.adapter.exists(this.file).then((exists) => {](https://github.com/meniam/obsidian-vocabulary-cards/blob/11d993d81717fe284e91ed3353d1d4fd29a767a4/src/CardStat.ts#L20)
Please use loadData() and saveData from the Obsidian API for storing your data (https://github.com/obsidianmd/obsidian-api/blob/249eb773347588f4e9fec7c600e1904cd9240fcb/obsidian.d.ts#L2982). This allows the data to be compatible with syncing platforms like Obsidian Sync, an example can be found in the sample plugin.

meniam commented:
@joethei, thank you for the review and your attention, it is very valuable.

I made all the changes according to the comments:

- fixed the license file;
- I use loadData to save data;
- deleted files: styles.css and main.js.
